### PR TITLE
chore(website): add april newsletter

### PR DESCRIPTION
**Pull request by TheSisb**
Add the April newsletter to the website.

### CODE REVIEW

File: packages/paste-website/src/pages/blog/2022-04-06-paste-newsletter.mdx [hunk 1]
```diff
@@ -0,0 +1,235 @@
+---
+title: Paste Design System Newsletter - April 2022 Edition
+slug: /blog/2022-04-06-paste-newsletter/
+date: 2022-04-06
+author: Paste
+avatar:
+author_link:
+status: published
+external_link:
+excerpt: If you're building a new platform team, your customers are going to go through many changes. In the case of a design system, tools, hacks, and practices that made your team wildly successful are also the cause of long term design and technical debt. Making corrective changes to mitigate these problems with a platform can often mean taking away these...
```
Comment (gloriliale) [inline packages/paste-website/src/pages/blog/2022-04-06-paste-newsletter.mdx:10]: todo: replace this text with some from the beginning of the newsletter

File: packages/paste-website/src/pages/blog/2022-04-06-paste-newsletter.mdx [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+<Paragraph>
+  <Button as="a" href="/" variant="primary">
+    Check out the Dark Theme
+  </Button>
+</Paragraph>
+
+### The Badge component enters Stage 2
+
+![The new and improved Badge component in Paste](../../assets/images/articles/2022-04-06-paste-newsletter/badge-2.png)
+A big [Badge component](/components/badge/) update improves the look in both the default and dark themes, as well as improving the interactivity for clickable Badges. We’ve also introduced “decorative” variants that can be used for attributes that don’t have a semantic meaning but would still benefit from the visual hierarchy of a Badge.
```
Comment (gloriliale) [inline packages/paste-website/src/pages/blog/2022-04-06-paste-newsletter.mdx:106]: Suggestion: there's inconsistent spacing for the images and the paragraphs after; I added some spaces to the ones that need it!

```suggestion
![The new and improved Badge component in Paste](../../assets/images/articles/2022-04-06-paste-newsletter/badge-2.png)

A big [Badge component](/components/badge/) update improves the look in both the default and dark themes, as well as improving the interactivity for clickable Badges. We’ve also introduced “decorative” variants that can be used for attributes that don’t have a semantic meaning but would still benefit from the visual hierarchy of a Badge.
```

File: packages/paste-website/src/pages/blog/2022-04-06-paste-newsletter.mdx [hunk 3]
```diff
@@ -0,0 +1,235 @@
...
+<Paragraph>
+  <Button as="a" href="/components/badge" variant="primary">
+    Check out the Badge component
+  </Button>
+</Paragraph>
+
+### Introducing the Filter Group pattern
+
+![Screenshot of a Filter Group pattern example](../../assets/images/articles/2022-04-06-paste-newsletter/filter-group-pattern-demo.png)
+Many of Twilio’s interfaces involve displaying data in tables.
```
Comment (gloriliale) [inline packages/paste-website/src/pages/blog/2022-04-06-paste-newsletter.mdx:117]:
```suggestion
![Screenshot of a Filter Group pattern example](../../assets/images/articles/2022-04-06-paste-newsletter/filter-group-pattern-demo.png)

Many of Twilio’s interfaces involve displaying data in tables.
```

File: packages/paste-website/src/pages/blog/2022-04-06-paste-newsletter.mdx [hunk 4]
```diff
@@ -0,0 +1,235 @@
...
+    Check out the Data Visualization Foundations
+  </Button>
+</Paragraph>
+
+## ⚡ Pastemates™ spotlight
+
+> Pastemates are individuals and teams who have been critical contributors to Paste, through advocating for adoption within their products and channeling important feedback and contributions back into the system.
+
+![Photo of this month's Pastemates. Left: Kate Hsiao. Center: Tess Hannel. Right: Zach Kuzmic.](../../assets/images/articles/2022-04-06-paste-newsletter/pastemates.png)
+We couldn’t keep it to just one Pastemate this time around; our Pastemates this release are Kate Hsiao, Tess Hannel,
```
Comment (gloriliale) [inline packages/paste-website/src/pages/blog/2022-04-06-paste-newsletter.mdx:149]:
```suggestion
![Photo of this month's Pastemates. Left: Kate Hsiao. Center: Tess Hannel. Right: Zach Kuzmic.](../../assets/images/articles/2022-04-06-paste-newsletter/pastemates.png)

We couldn’t keep it to just one Pastemate this time around; our Pastemates this release are Kate Hsiao, Tess Hannel,
```

**Review by shleewhite — APPROVED** [review]
🚢

**Review by gloriliale — APPROVED** [review]
Woohoo new blog post!

**Pull request merged**